### PR TITLE
Added Non-traffic violation data

### PR DESCRIPTION
**Pull request by ghost**
Alright, added the non-traffic violation data. Feedback please?

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -180,6 +180,13 @@
             iconSize: [32, 32],
             iconAnchor: [16, 32],
             popupAnchor: [0, -16]
+        }),
+        NON_TRAFFIC_VIOLATION: L.divIcon({
+            className: 'map-pin darkorchid',
+            html: '<i class="fa fa-road"></i>',
```
Comment (vonbearshark) [inline main.js]: These are actually _non_ traffic violations, so I don't think a road is fitting. It's a hard one to find an icon for, for sure. What about something generic like this: http://fontawesome.io/icon/sticky-note-o/? So it kind of seems like a citation. We will probably change them all eventually, anyway

**Review by vonbearshark — COMMENTED** [review]
Great job! Could we change the icon? Other than that this looks great

**Review by vonbearshark — APPROVED** [review]
Looks great!

**Comment by ghost** [comment]
Cool! Is there anything similar or beginner-ish that I can work on?

**Comment by vonbearshark** [comment]
I'll take a look and mention you in a comment. Do you know Python/Flask?

**Comment by ghost** [comment]
Not too well but i'm looking to learn!

**Comment by vonbearshark** [comment]
Maybe @RitwikGupta can find something needed in the PittAPI repo: https://github.com/Pitt-CSC/PittAPI. I'll keep you posted

**Comment by vonbearshark** [comment]
Does this one interest you? https://github.com/Pitt-CSC/PittAPI/issues/10

**Comment by ghost** [comment]
Sure! I'll probably be asking a lot of questions but i'll give it a shot.

**Pull request merged**